11_bank_card.py

```diff
@@ -0,0 +1,154 @@
+"""
+Hometask 21 Bank card operation.
+
+# write class for bank card.
+# Class should reflect card lifecycle, card operations etc.
+# use CVV, PIN, Name, Surname , end date, card_num as initial params
+# class should have in addition to common logic some class attributes,
+# as minimum one classmethod and
+# as minimum  one staticmethod, two or more getters/setters,
+# __str_ magic method
+# do not forget about block
+
+"""
+
+import datetime
+import random
+
+
+class BankCard:
+    """Make class."""
+
+    bank = 'Best Bank'
+    card = 'VISA'
+
+    def __init__(self, name, surname, balance):
+        """Make init."""
+        self.name = name
+        self.surname = surname
+        self._balance = balance
+        self.__card_num = self.get_card_number
+        self._exp_date = self.exp_date()
+        self._card_pin = self.card_pin()
+        self._card_cvv = self.card_cvv()
+
+    @staticmethod
+    def get_card_number():
+        """secret_card = f"{'*' * 12}{card[-4:]}."""
+        """card = str(random.randint(1000000000000000, 9999999999999999))
+        it's also works."""
+        """card = str(random.randint(10 ** 15, (10 ** 16) - 1))"""
+        """f"{'*' * 12}{card[-4:]}" """
+        return str(random.randint(10 ** 15, (10 ** 16) - 1))
+
+    @staticmethod
+    def exp_date():
+        """Find exp date."""
+        today = datetime.date.today()
+        exp_month = today.month
+        exp_year = int(str(today.year + 4)[2:])
+        return f'{exp_month:02d}/{exp_year}'
+
+    @staticmethod
+    def card_pin():
+        """Make static method."""
+        card_pin = random.randint(1000, 9999)
+        return card_pin
+
+    @staticmethod
+    def card_cvv():
+        """Find random cvv code."""
+        return random.randint(100, 999)
+
+    def get_card_cvv(self):
+        """Getter."""
+        return self._card_cvv
+
+    def set_card_cvv(self, value):
```

Comment:
This better to do via @property  for example
```
@property
    def pin(self):
        return self.__pin

@pin.setter
def pin(self, value):
    self.__pin = value 
```
Please change all your setters and getters to this schema